### PR TITLE
Version 0.1.0 of the Schema-Registry-UI

### PR DESCRIPTION
**Pull request by cmarquezrusso**
**What this PR does / why we need it**:
This PR adds a new Chart into the incubator. 
Schema-Registry UI is a web tool for the confluentinc/schema-registry in order to create / view / search / evolve / view history & configure Avro schemas of your Kafka cluster.

More info here: https://github.com/Landoop/schema-registry-ui

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by cristian04** [comment]
Hey /assign @prydonius, could you please review this? Thanks :)

**Comment by timstoop** [comment]
Could you please add an OWNERS file as well? That's the only thing that I think is missing. The rest is nice and simple and looks ok.

**Comment by timstoop** [comment]
Heh you added yourself to the repo owners, that's not what I meant. Please revert that.

I requested that you add an OWNERS file to this Chart. Check https://github.com/kubernetes/charts/blob/master/stable/cert-manager/OWNERS for an example.

**Comment by cmarquezrusso** [comment]
Done. Sorry for the mistake regarding the OWNERS file.

/joke

File: incubator/schema-registry-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Schema-Registry-UI Helm Chart
+This helm chart creates a [Schema-Registry-UI server](https://github.com/Landoop/schema-registry-ui).
+
+This is a web tool for the confluentinc/schema-registry in order to create / view / search / evolve / view history & configure Avro schemas of your Kafka cluster.
+
+
+## Prerequisites
+* Kubernetes 1.6
```
Comment (unguiculus) [inline incubator/schema-registry-ui/README.md]: 1.8+

File: incubator/schema-registry-ui/OWNERS [hunk 2]
```diff
@@ -0,0 +1,25 @@
+approvers:
```
Comment (unguiculus) [inline incubator/schema-registry-ui/OWNERS]: Don't put all the chart maintainers in there. You should also add yourself as approver. However, for the mechanism to work, you'd have to become a member of the Kubernetes org on Github.

See https://github.com/kubernetes/charts#owning-and-maintaining-a-chart

File: incubator/schema-registry-ui/README.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+You can install the chart with the release name `srui` as below.
+
+```console
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install --name srui incubator/schema-registry-ui
+```
+
+If you do not specify a name, helm will select a name for you.
+
+### Installed Components
```
Comment (unguiculus) [inline incubator/schema-registry-ui/README.md]: I'd remove this section. `helm status <release>` is probably the better choice anyways.

**Review by unguiculus — CHANGES_REQUESTED** [review]
I'd suggest you move the chart to stable directly.

File: stable/schema-registry-ui/OWNERS [hunk 4]
```diff
@@ -0,0 +1,11 @@
+approvers:
```
Comment (unguiculus) [inline stable/schema-registry-ui/OWNERS:1]: Again, only maintainers of this chart should be in the file, which is you only.

File: stable/schema-registry-ui/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+keywords:
+  - confluent
+  - schema-registry-ui
+  - Kafka
+  - avro
+  - schema-registry
+version: 0.1.0
+maintainers:
+- email: cristian04@gmail.com
+  name: Cristian Russo
```
Comment (unguiculus) [inline stable/schema-registry-ui/Chart.yaml]: Use Github username

**Pull request merged**